openmrs-19/src/test/java/org/motechproject/openmrs19/it/MRSPatientServiceIT.java

```diff
@@ -251,6 +273,18 @@ private void prepareConceptOfDeath() throws ConceptNameAlreadyInUseException {
         causeOfDeath = conceptAdapter.getConceptByUuid(DEFAULT_CONFIG_NAME, uuid);
     }
 
+    private List<Identifier> prepareIdentifier() {
+        List<Identifier> testList = new ArrayList<>();
```

Comment:
please add motechId here, because in OpenMRS module the MOTECH ID identifier is required for adding patient